cmdgad

```diff
@@ -1,5 +1,5 @@
-#!/usr/bin/python -B
-#!/usr/bin/env python -B
+#!/home/jzh/anaconda2/bin/python
```

Comment:
Have taken care of it...seems no need to emphasize which python distribution to use...